docs/manage/ml/train-model.md

```diff
@@ -14,13 +14,14 @@ You can tag [images collected](../../../services/data/configure-data-capture/) b
 
 {{<youtube embed_url="https://www.youtube-nocookie.com/embed/CP14LR0Pq64">}}
 
-## Label a dataset
```

Comment:
Above, the sentence that starts with "You can tag images collected.." should say "use the annotated data to train a **Single Label Classification Model**, **Multi Label Classification Model**, or **Object Detection Model** "